frontend/src/core/comments/components/AddComment.js

```diff
@@ -67,8 +79,10 @@ export default function AddComments(props: Props) {
                     name='comment'
                     dir='auto'
                     placeholder={ `Write a comment…` }
+                    rows={ minRows }
                     ref={ commentInput }
-                    onKeyDown={ onEnterSubmit }
+                    onChange={ handleOnChange }
+                    onKeyUp={ submitComment }
```

Comment:
I believe you should use `onKeyDown` here, same as we did before. Otherwise the newline (`%0A`) character enters the `<textarea>` before we call `submitComment()`, which means `commentInput.current.value` isn't empty.

Reply:
Ahhh, yes that makes sense.  I made the change and it indeed fixed the issue.  Thanks!